test/spoom/context/sorbet_test.rb

```diff
@@ -339,6 +341,25 @@ def test_context_srb_files_with_custom_ignore
         context.destroy!
       end
 
+      def test_context_srb_files_with_custom_ignore_prefixed_with_slash
+        context = Context.mktmp!
+
+        context.write_sorbet_config!(<<~CONFIG)
+          .
+          --ignore=/config/routes.rb
+          --ignore=/lib/generators/
+        CONFIG
+
+        context.write!("foo.rb", "")
+        context.write!("config/routes.rb", "")
+        context.write!("lib/generators/generator1.rb", "")
+        context.write!("lib/generators/generator2.rb", "")
```

Comment:
Per my comment above, what happens if we add a file named `/foo/lib/generators/bar.rb` here?

Reply:
I added this test case 👍